doc/languages-frameworks/python.section.md

```diff
@@ -734,6 +734,51 @@ work in any of the formats supported by `buildPythonPackage` currently,
 with the exception of `other` (see `format` in
 [`buildPythonPackage` parameters](#buildpythonpackage-parameters) for more details).
 
+##### Using sphinxHook {#using-sphinxhook}
+
+The `sphinxHook` is a helpful tool to build documentation and manpages
+using the popular Sphinx documentation generator.
+It is setup to automatically find common documentation source paths and
+render them using the default `html` style.
+
+```
+  outputs = [
+    "doc"
+  ];
```

Comment:
```suggestion
```nix
  outputs = [
    "out"
    "doc"
  ];
```

we should add the out output so that people do not forget it and things break.